### ISSUE TITLE
Hybridauth\User\Profile::$identifier typed as int|null but string given

### ISSUE THREAD

**Issue by KevsRepos** [hybridauth/hybridauth#1375]
## Bug, feature or question?

Is this a bug, a feature request or a question?
Give us a short description.

The property `Hybridauth\User\Profile::$identifier` is typed with phpdoc as `int|null` but the actual returned value is a `string`. This gets in trouble with php linters.

### Version and provider
What version of Hybrid Auth does this relates to?
3.8

Is this an issue with a provider? If yes, which one?
Found this while working with the  google provider only.

### Reproduction
How can we replicate this issue?

```php
$hybridauth = new Hybridauth($config);

$adapter = $hybridauth->authenticate($providerName);

$userProfile = $adapter->getUserProfile();

$userProfile->identifier; // will return string
```

**Comment by ApacheEx** [comment]
good catch. I think we can try to extend the type for this field. So, at then end it will be `string|int|null`


PR is welcome
